recipes/sagemaker-code-editor/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "sagemaker-code-editor" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz
+  sha256: 9effda827c45e1fd8dfc534ce310791ee46070d818ede89abf19e3ec9c8906d0
+  folder: sagemaker-code-editor
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - sysroot_linux-64 ==2.17
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+    - nodejs >=18.15,<19
```

Comment:
Add a comment on why this is needed.

Reply:
In latest commit, added a comment for nodejs range - it's needed to build microsoft/vscode code - https://github.com/microsoft/vscode/wiki/How-to-Contribute